Configuration/TypoScript/Library/themes.basic.constantsts

```diff
@@ -81,3 +81,8 @@ themes.configuration.contacts.email = info@info.se
 
 # cat=theme,basic/customPath; type=string; label= Custom Path
 themes.configuration.customPath = fileadmin/custom
+
+# customsubcategory=maps= Google map configuration
+
+# cat=theme,basic/maps; type=string; label= Google map api key
```

Comment:
I think we need to put this constant into "advanced" tab in t3kit themes.
`# cat=theme,advanced/maps; type=string; label= Google map api key`